tests/comparison_filter/076.phpt

```diff
@@ -0,0 +1,57 @@
+--TEST--
```

Comment:
Good test, but could you perhaps move it to another directory? I'd suggest we keep our own tests separate from the comparison tests. Mostly so that the tests in the `comparison_*` directories are in sync with the [JSONPath Comparison](https://cburgmer.github.io/json-path-comparison/) ones, in case we join that club.

Reply:
Sure, no problem